bitnami/solr/values.yaml

```diff
@@ -0,0 +1,387 @@
+## Global Docker image parameters
+## Please, note that this will override the image parameters, including dependencies, configured to use the global value
+## Current available global Docker image parameters: imageRegistry and imagePullSecrets
+##
+# global:
+#   imageRegistry: myRegistryName
+#   imagePullSecrets:
+#     - myRegistryKeySecretName
+#   storageClass: myStorageClass
+
+## Add labels to all the deployed resources
+##
+commonLabels: {}
+
+## Add annotations to all the deployed resources
+##
+commonAnnotations: {}
+
+## Kubernetes Cluster Domain
+##
+clusterDomain: cluster.local
+
+## Extra objects to deploy (value evaluated as a template)
+##
+extraDeploy: []
+
+## Number of nodes
+##
+replicaCount: 1
+
+service:
+  type: LoadBalancer
+  # HTTP Port
+  port: 8983
+  # HTTPS Port
+  httpsPort: 8983
+  ## loadBalancerIP for the SuiteCRM Service (optional, cloud specific)
+  ## ref: http://kubernetes.io/docs/user-guide/services/#type-loadbalancer
+  ##
+  ## loadBalancerIP:
+  ##
+  ## nodePorts:
+  ##   http: <to set explicitly, choose port between 30000-32767>
+  ##   https: <to set explicitly, choose port between 30000-32767>
+  nodePorts:
+    http: ""
+    https: ""
+  ## Enable client source IP preservation
+  ## ref http://kubernetes.io/docs/tasks/access-application-cluster/create-external-load-balancer/#preserving-the-client-source-ip
+  ##
+  externalTrafficPolicy: Cluster
+
+## %%MAIN_CONTAINER/POD_DESCRIPTION%%
+##
+solr:
+  enabled: true
```

Comment:
As this application only has one component (Solr), there's no need to create a `solr` section in the values. This is useful when there are multiple components, for example, in a database that has some `primary` nodes and some `secondary` nodes.

Please, remove the `solr` section and leave the values in the root of the document (so, instead of `.Values.solr.image` it would be `.Values.image`

Reply:
Done!